DigitalExcellence/dex-frontend
# Feature/536 update edit page

## Description

Updated the Edit page according to issue #536 , while also fixing #561 

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Go to the project overview page and click on some filters
2. Click on a project that you can edit
2. Click on the edit button
3. See the new and improved edit page 😎 
4. Go trough all the steps to see everything works.
5. Save your changes to see the detail page (& overview in the background) get updated after saving 🎈 
6. Click on some filters to see that they don't get removed after editing. 
🦸  **Bonuspoints:** Check same thing in different browsers and mobile 😄 

## Link to issue
Closes: #561 
Closes: #536
